main.md

```diff
@@ -713,6 +820,24 @@ trusting/using any of the contents of an SD-JWT:
 
 If any step fails, the input is not valid and processing MUST be aborted.
 
+## Processing Model {#processing_model}
+
+Neither an SD-JWT nor an SD-JWT-R is suitable for direct use by an application.
+Besides the REQUIRED verification steps listed above, it is further RECOMMENDED
+that tan application-consumable format is generated from the data released in
+the SD-JWT-Release. The RECOMMENDED way is to merge the released claims and any
+plaintext claims in the SD-JWT recursively:
+
+ * Objects from the released claims must be merged into existing objects from the SD-JWT. 
+ * If a key is present in both objects:
+   * If the value in the released claims is and object and the value in the
+     SD-JWT claims is an object, the two objects MUST be merged recursively.
+   * Else, the value in the released claims MUST be used.
+
+The keys `sd_digests` and `sd_hash_alg` SHOULD be removed prior to further
+processing. 
+
+An example can be seen in Example 3 in the Appendix.
```

Comment:
why not an example with simple claim structure but complex claim structure?

Reply:
The complex claim structure example shows all interesting cases of the merging, especially how to merge objects. We can additionally show a simple example if you like.